### PR TITLE
AAA pattern update for Dresses and Tshirts

### PR DESCRIPTION
**Pull request by MCLRDRI15**

### CODE REVIEW

File: cypress/integration/dresses-list.spec.ts [hunk 1]
```diff
@@ -1,15 +1,28 @@
...
+    menuContentPage = new MenuContentPage();
+    dressesListPage = new DressesListPage();
+  });
 
-//   it("show the available dresses", () => {
-//     // ... realiza la prueba
-//   });
-// });
+  it("show the available dresses", () => {
+    // ... realiza la prueba
```
Comment (manuelq12) [inline cypress/integration/dresses-list.spec.ts]: You can erase this comment

File: cypress/integration/dresses-list.spec.ts [hunk 2]
```diff
@@ -1,15 +1,28 @@
...
+
+    // Arrange
+    menuContentPage.visitMenuContentPage();
+    const allDressesNames =
+     ["Printed Dress", "Printed Dress", "Printed Summer Dress", "Printed Summer Dress", "Printed Chiffon Dress"];
```
Comment (manuelq12) [inline cypress/integration/dresses-list.spec.ts]: My suggestion is that this const array can be part of the before with the initialization of the pages, i think its better placement (is a suggestion)

Comment (manuelq12) [inline cypress/integration/dresses-list.spec.ts:18]: My suggestion is that this const array can be part of the before with the initialization of the pages, as they are not variables but constants

File: cypress/integration/buy-shirt.spec.ts [hunk 3]
```diff
@@ -17,46 +17,28 @@ const paymentStepPage = new PaymentStepPage();
 
 describe("Buy a t-shirt", () => {
   it("then the t-shirt should be bought", () => {
```
Comment (manuelq12) [inline cypress/integration/buy-shirt.spec.ts:29]: You should place the intialization of the pages with a before hook as it was shown for the dresses-list-page.spec

Comment (manuelq12) [inline cypress/integration/buy-shirt.spec.ts:19]: You should place the intialization of the pages with a before hook as it was shown for the dresses-list-page.spec

File: cypress/integration/dresses-list.spec.ts (hunk 2 above)
Comment (MCLRDRI15) [reply cypress/integration/dresses-list.spec.ts]: Sure Manuel Thanks !

**Review by manuelq12 — APPROVED** [review]

**Comment by manuelq12** [comment]
LGTM 🎖️

**Pull request merged**